cnxml/jing.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+import subprocess
+
+from .util import lookup_resource
+
+__all__ = (
+    'jing',
+)
+
+
+JING_JAR = lookup_resource('jing.jar')
+
+# This will map fatal messages to something human readable.
+KNOWN_FATAL_MESSAGES_MAPPING = {
+    'exception "java.io.IOException" thrown: Stream closed.':
+    'DOCTYPE declaration not allowed',
```

Comment:
Yikes, this is really the error it throws for DOCTYPE? Perhaps stale-ware is right.